src/API/Helpers/RequestBuilder.php

```diff
@@ -286,6 +286,20 @@ public function withParams($params) {
         return $this;
     }
 
+    /**
+     * Add URL parameters using $key => $value array.
```

Comment:
isn't this a dict?

Reply:
@lbalmaceda - Yes, just being explicit. There's another function that looks for an array of arrays with `key` and `value` keys